_includes/shared/particular_data.md

```diff
@@ -57,6 +57,7 @@ In general you should get started with the [tutorials](/tutorial). The pages tha
 - [Getting started with data from the BIOIMAGE suite](/getting_started/bioimage)
 - [Getting started with data from AnyWave](/getting_started/anywave)
 - [Getting started with data from Homer](/getting_started/homer)
+- [Getting started with data from Seg3D](/getting_started/seg3d)
```

Comment:
I think we should change this for Seg3D and AnyWave, since we are not really getting started with data that originates from them (whereas with Homer and BIOIMAGE we do import "their" data).